packages/bun-types/bun.d.ts

```diff
@@ -2304,7 +2306,7 @@ declare module "bun" {
     unix: string;
     tls?: TLSOptions;
   }
-  interface Errorlike extends Error {
+  interface ErrorLike extends Error {
```

Comment:
I agree with this rename, but let's make sure we maintain an alias somewhere that is deprecated.
This goes for anything we remove, like `AnyFunction`. We should not fully remove.

Can we add a `deprecated.d.ts` file?

Reply:
Oh right, that was just changed because of cSpell marking it as an error... we can yes add an alias and mark it as deprecated for now.

Aboud adding a file for deprecated features... it might yes be a good idea since i have seen some other stuff also marked as deprecated